reconcile/gitlab_owners.py

```diff
@@ -325,6 +325,7 @@ def act(repo, dry_run, instance, settings, defer=None):
             merge_request=mr,
             owners=project_owners,
             dry_run=dry_run,
+            persistent_lgtm=repo["gitlabRepoOwners"]["persistentLgtm"] or False,
```

Comment:
Can `repo["gitlabRepoOwners"]` be `None`?

Reply:
I don't think so, but anyway, I changed it in 566deed